invenio_stats/aggregations.py

```diff
@@ -361,19 +388,24 @@ def agg_iter(self, dt):
                     else:
                         aggregation_data[destination] = source(doc, aggregation_data)
 
-                index_name = "stats-{0}-{1}".format(
-                    self.event, interval_date.strftime(self.index_name_suffix)
+                index_name = prefix_index(
+                    "stats-{0}-{1}".format(
+                        self.event, interval_date.strftime(self.index_name_suffix)
+                    )
                 )
-
+                events += 1
                 yield {
                     "_id": "{0}-{1}".format(
                         aggregation["key"], interval_date.strftime(self.doc_id_suffix)
                     ),
-                    "_index": prefix_index(index_name),
+                    "_index": index_name,
                     "_source": aggregation_data,
                 }
+        # It might be good to put some more logging on the services, either with this or log.info, log.debug...
```

Comment:
This print can also be removed if you think that it is too much (or if there is a better way of logging these messages)